src/components/Menus/MainMenu/useMainMenuContent.tsx

```diff
@@ -248,6 +251,28 @@ export const useMainMenuContent = () => {
         openInNewTab(DISCORD_URL);
       },
     },
+    {
+      label: `${pathname.includes('memecoins') ? 'Go normal mode' : 'Go degen mode'}`,
+      prefixIcon: <WhatshotIcon />,
+      showMoreIcon: false,
+      onClick: () => {
+        trackEvent({
+          category: TrackingCategory.Menu,
+          label: 'click-degen-mode-link',
+          action: TrackingAction.ClickJumperLearnLink,
```

Comment:
Adjust that one, create a new TrackingAction within TrackingKeys file

Reply:
you are right, thanks.